content/en/docs/concepts/deployment-models/index.md

```diff
@@ -0,0 +1,421 @@
+approach helps limit the potential problems caused by incorrect configurations.
+
+In Istio, you can configure a single service mesh to span any number of
+clusters.
+
+### Single cluster
+
+In the simplest case, you can confine an Istio mesh to a single cluster. A
+single cluster usually operates over a [single network](#single-network), but it
+varies between infrastructure providers. In single cluster and single network
+models, you commonly deploy a single Istio control plane, which results in the
+simplest Istio deployment.
+
+{{< image width="50%"
+    link="./single-cluster.svg"
+    alt="A service mesh with a single cluster"
+    title="Single cluster"
+    caption="A service mesh with a single cluster"
+    >}}
+
+Single cluster deployments offer simplicity, but lack fault isolation and
+failover. If you need higher availability, you should use multiple clusters.
+
+### Multiple clusters
+
+You can configure your mesh to include multiple clusters; such configurations
+are known as **multi-cluster**. Using multiple clusters
+affords the following capabilities beyond that of a single cluster model:
+
+- Fault isolation and failover: `cluster-1` goes down, failover to `cluster-2`.
+- Location-aware routing and failover: Send requests to the nearest service.
+- Configuration isolation: A wrong configuration only affects a single cluster.
+- Various [control plane models](#control-plane-models): Support different
+  levels of availability.
+- Team or project isolation: Each team runs its own set of clusters.
+
+{{< image width="50%"
+    link="./multi-cluster.svg"
+    alt="A service mesh with multiple clusters"
+    title="Multi-cluster"
+    caption="A service mesh with multiple clusters"
+    >}}
+
+Multi-cluster deployments give you a greater degree of isolation and
+availability but increase complexity. If your systems have high availability
+requirements, you likely need clusters across multiple zones and regions. You
+can canary configuration changes or new binary releases in a single cluster,
+where the configuration changes only affect a small amount of user traffic.
+Additionally, if a cluster has a problem, you can temporarily route traffic to
+nearby clusters until you address the issue.
+
+You can configure inter-cluster communication based on the
+[network](#network-models) and the options supported by your cloud provider. For
+example, if two clusters reside on the same underlying network, you can enable
+cross-cluster communication by simply configuring firewall rules.
+
+## Network models
+
+Many production systems require multiple networks or subnets for isolation
+and high availability. Istio supports spanning a service mesh over a variety of
+network topologies.
+
+### Single network
+
+In the simplest case, a service mesh operates over a single fully
+connected network. A single network model satisfies the following conditions:
+
+- The network consists of a number of connected subnets without any
```

Comment:
> The network consists of a number of connected subnets without any overlap in IP or VIP ranges.

Isn't it a technical detail, irrelevant to the users who are not interested to know how the next item is achieved? Suppose I am a user and I order several clusters from my cloud provider. I am interested to know if workload instances from one cluster can reach workload instances in other clusters. Why should I understand subnets, IP and VIP ranges? I am sure this is an interesting information for networking people, but is it necessary for all the Istio users?

Reply:
Agreed.

Let's drop the discussion of subnets altogether ... it really is an implementation detail. Let's just focus on a "network" in terms of reachability.

Reply:
@john-a-joyce had strong feeling about it if I remember correctly. Would it be ok to drop the subnets as suggested?